code/modules/ai/ai_behaviors/human_mobs/hazard_avoidance.dm

```diff
@@ -0,0 +1,112 @@
+/datum/ai_behavior/human
+	///Mob tries to keep away from hazards
+	var/avoid_hazards = TRUE
```

Comment:
do wonder if this is something useful for xvh ai

Reply:
may be opee though?

Reply:
It would probs make em worse, since a flamer could just hold back the horde if they're no longer fearless.
I'm trying to keep this PR to just the new AI type though, in another PR I will bring all the useful/relevant stuff to the xeno side